R/lump.R

```diff
@@ -137,6 +185,54 @@ fct_lump_min <- function(f, min, w = NULL, other_level = "Other") {
 
 }
 
+.fct_lump_min <- function(min, count, levels, other_level) {
```

Comment:
I think pulling these functions out make the code harder to understand because you've added an additional layer of indirection — I don't think it's worth it given that they're each only called from one place.

Reply:
`.fct_lump_prop` and `.fct_lump_n` are called from both `fct_lump` and their distinct versions, and I extended that pattern for `.fct_lump_min` and `.fct_lump_count` which aren't in `fct_lump` - would you prefer to keep `fct_lump(prop)` as well as `fct_lump_prop` or move the internals into their own functions with a clear separation?